interface/main/dated_reminders/dated_reminders.php

```diff
@@ -88,9 +88,10 @@
       </style>
       <script type="text/javascript">
          $(document).ready(function (){
-            <?php if (!$hasAlerts) {
-                echo '$(".hideDR").html("<span>'.xla('Show Reminders').'</span>"); $(".drHide").hide();';
-} ?>
+            <?php
+            if (!$hasAlerts) {
+              // echo '$(".hideDR").html("<span>'.xla('Show Reminders').'</span>"); $(".drHide").hide();';
```

Comment:
odd to have a conditional with only a comment in it

Reply:
This was the default behavior on the original Message Center. I wanted to get rid of this behavior because it is not necessary as  tabs view is used to hide the Dated Reminders.  If you are OK with my restyling that code can be removed.

Reply:
Lets remove it :)